lib/web/apiserver_test.go

```diff
@@ -161,11 +161,17 @@ func (s *WebSuite) SetUpTest(c *C) {
 	s.user = u.Username
 	s.clock = clockwork.NewFakeClock()
 
+	networkingConfig, err := types.NewClusterNetworkingConfigFromConfigFile(types.ClusterNetworkingConfigSpecV2{
+		KeepAliveInterval: types.Duration(time.Second * 10),
```

Comment:
nit: 10 * Second, so  it reads more naturally?

```suggestion
		KeepAliveInterval: types.Duration(10 * time.Second),
```